### PR TITLE
[FEATURE] Add ignore file paths to directory mapping

### PR DESCRIPTION
**Pull request by TopHatCroat**
This will allow to ignore some of the files.
This is useful, for example, when you want to keep your test files in the same directory as the source file, hence parsing the test spec files would break the regular build.

Also, allow to change default security options

### CODE REVIEW

**Comment by Cody2333** [comment]
Thanks~ , could you please create your PR to the develop branch ?

File: example/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const common = {
   baseUrl: 'http://localhost:3000',
-  port: 3000
+  port: 5000
```
Comment (Cody2333) [inline example/config.js]: why modify the port?

Comment (TopHatCroat) [reply example/config.js]: It was by mistake, I undid that

**Comment by TopHatCroat** [comment]
Rebased.
Note to self: RTFM

**Pull request merged**